apps/example-todo-app/next.config.js

```diff
@@ -1,15 +1,20 @@
 // This file changes the routing to allow top-level prefixes
 
-module.exports = {
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
   async rewrites() {
     return {
       beforeFiles: [
         // Nextjs by default requires a /api prefix, let's remove that
-        {
-          source: "/:path*",
-          destination: "/api/:path*",
-        },
+        // REVIEW: I was not able to make the redirect work
+        // {
+        //   source: "/:path*",
+        //   destination: "/api/:path*",
+        // },
```

Comment:
@itelo I think it would be a lot better if we created an `example-edge-todo-app` instead of modifying the existing one, we can merge them later, but for now changing this file is too difficult because it's not clear if you're introducing a regression